src/lib/openapi/index.ts

```diff
@@ -38,10 +65,33 @@ export const createOpenApiSchema = (
                 createStrategySchema,
                 featureSchema,
                 featuresSchema,
+                featureEnvironmentInfoSchema,
+                featureToggleDtoSchema,
+                emptyResponseSchema,
                 overrideSchema,
                 parametersSchema,
+                patchFeatureAddOperationSchema,
+                patchFeatureRemoveOperationSchema,
+                patchFeatureReplaceOperationSchema,
+                patchFeatureMoveOperationSchema,
+                patchFeatureCopyOperationSchema,
+                patchFeatureTestOperationSchema,
+                patchFeatureGetOperationSchema,
+                patchFeatureOperationSchema,
+                patchStrategyAddOperationSchema,
+                patchStrategyRemoveOperationSchema,
+                patchStrategyReplaceOperationSchema,
+                patchStrategyMoveOperationSchema,
+                patchStrategyCopyOperationSchema,
+                patchStrategyTestOperationSchema,
+                patchStrategyGetOperationSchema,
+                patchStrategyOperationSchema,
```

Comment:
I think we can start a bit simpler here, with just a few small generic schemas for add/remove/replace/move/copy, and reuse those across domain types.

The value fields of the operations could be something simple like `string | number`. I don't think we should to use the whole feature/strategy schema as values at least, since the request operates on a specific value at some path.

I don't think we use test/get so those can probably be removed.